lib/src/builder/builder.dart

```diff
@@ -33,88 +25,93 @@ class OverReactBuilder extends Builder {
       return;
```

Comment:
I'm not sure how feasible it is without actually examining the source of the part files, but it would be nice to get a warning as a consumer if I've got a parent library file whose parts contain over_react components, but which is missing the `*.over_react.g.dart` part directive.

Reply:
Addressed in 93d4593